### DISCUSSION TITLE
1.2.7 stable

### DISCUSSION THREAD

**Discussion by trueromanus** [anilibria/anilibria-winmaclinux#137] [Анонсы]
- Главное меню теперь поддерживает прокрутку
- Некоторые эффекты (тени под элементами, постеры) могут выглядеть немного не так как было. Это изменения в рамках подготовки к поддержке Qt6
- Кастомный тулбар теперь встроен в окно
- Добавлена история переходов
- Настройка "Сохранять выбранный плеер" на странице Видеоплеер
- Показывать признак добавления в избранное на странице Связанные релизы
- Добавлена кастомизация выпадающего меню и переключателя в Менеджере Тем

<hr /><em>This discussion was created from the release <a href='https://github.com/anilibria/anilibria-winmaclinux/releases/tag/1.2.7'>1.2.7 stable</a>.</em>

**Comment by Karpach0** [comment]
А можно сделать версию без трояна, а то у меня антивирус ругаеца

**Reply by trueromanus** [reply]
Какой антивирус и на что ругается?

**Reply by Mindubaev** [reply]
Avast Premium  определяет TorrentStream как FileRepMalware. Так же приложение в последние дни перестало синхронизироваться с сервером (у Anilibrix такой проблемы не наблюдается).

**Reply by trueromanus** [reply]
@Mindubaev Синхронизацию можно починить сменой сервера для этого перейдите на страницу "Обслуживание", нажмите на кнопку "изменить адрес api" и выберите в поле "выбрать сервер" значение wwnd. Далее перезапустите приложение.

"Avast Premium определяет TorrentStream как FileRepMalware." - да уже проверили он и 360 TS ругаются, остальные антивирусы молчат или помечают как подозрительную активность, причем этот файл включен в 3 версии подряд и до этого судя по всему он на него не ругался. Поэтому решать исключительно Вам что с этим делать.

**Reply by Mindubaev** [reply]
@trueromanus Спасибо за совет, удалось синхронизироваться.

**Reply by Karpach0** [reply]
360 ругался, спасибо что исправили
